app/models/gi_bill_feedback.rb

```diff
@@ -20,6 +34,50 @@ def get_user_details
     { 'profile_data' => profile_data }
   end
 
+  def remove_malformed_options(options_hash)
+    return_val = {}
+    options_hash.each do |k, v|
+      next if FEEDBACK_MAPPINGS.keys.include?(k)
+
+      return_val[k] = v
+    end
+
+    return_val
+  end
+
+  def transform_malformed_options(options_hash)
+    return_val = {}
+
+    options_hash.each do |k, v|
+      FEEDBACK_MAPPINGS[k].tap do |new_key|
+        if new_key.blank?
+          return_val[k] = v
+        else
+          return_val[new_key] = v
+        end
+      end
+    end
+
+    return_val
+  end
+
+  def fix_options(options_hash, key)
+    # if user saves form with options, then goes back to the page
+    # and fills out options again, then the hash will contain both malformed and
+    # normal option keys
+    keys_size = options_hash.keys.size
+
+    %w[programs assistance].each do |attr|
```

Comment:
Is there a reason we're checking the size for both `programs` and `assistance`, if we already know which one we're dealing with? Could this just be something like
```
max_size = VetsJsonSchema::SCHEMAS[FORM_ID]['properties'][
        'educationDetails'
      ]['properties'][key]['properties'].size

return remove_malformed_options(options_hash) if keys_size > max_size
```
without looping through `%w[programs assistance]`?